index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="styles/styles.css" />
+  <script defer src="scripts/index.js"></script>
+  <title>Haker_news_JS</title>
+</head>
+<body>
+  <main class="page">
+    <header class="page__header top-panel">      
+      <h1 class="logo__title">Hacker News</h1>
+      </header>    
+      <section class="page__news">
+        <ol id="news-list" class="generic-list">
+        </ol>
+      </section>
+      <aside class="page__form">
+        <form class="form" name="user">
+          <h2 id="loginFormTitle" class="form__title">Login form</h2>
+            <div class="form__control field">
+            <label for="nameField" class="field__label">User name</label>
+            <input id="nameField" type="text" class="field__input" placeholder="name" name="name">
+          </div>
+          <div class="form__control field">
+            <label class="field__label">Gender
+            </label>
+            <div id="genderField" class="form__radio radio">
```

Comment:
it seems radio is a state of the field block:
```<div id="genderField" class="form__radio field">```
```<input class="field__input field__input--radio" type="radio"```

Reply:
Fixed